isolate_example/README.md

```diff
@@ -0,0 +1,23 @@
+# Isolate Example
+
+A sample application that demonstrate best practices when using [`isolates`](https://api.dartlang.org/stable/2.3.1/dart-isolate/Isolate-class.html).
+
+## Goals
+
+* Display the performance benefits of isolates when using them in the right situation.
+* Show how to use the `compute` method for straightforward computations.
+* Demonstrate how to initialize and use an isolate.
+* Show the cost of moving data between isolates and provide alternatives .
```

Comment:
Looks like there's an extra space between "alternatives" and the period at the end of the sentence. Also, you can remove one of the two lines of whitespace after this block.